README.md

```diff
@@ -37,27 +37,25 @@ This Widget uses the ForgeRock JavaScript SDK internally. It adds a UI rendering
 
 This Widget can be used in two different ways (or "form factors"):
 
-1. Modal component: this renders the form elements inside a modal dialog that can be opened and closed. This component is mounted outside of your app's controlled DOM.
-2. Inline component: this is just the form elements themselves, no container. This component is intended to be rendered inside your app's controlled DOM.
+1. **Modal** component: this renders the form elements inside a modal dialog that can be opened and closed. This component is mounted outside of your app's controlled DOM.
```

Comment:
Maybe we can give an example to some effect like `if react is rendered inside a `<div id="my-app"></div>` then the modal would be rendered outside of this `div`. Just to emphasize it more.

Reply:
Sorry the formatting removed some code which makes this make no sense. sent on slack